packages/e2e/pages/profile.page.ts

```diff
@@ -0,0 +1,21 @@
+import { By } from 'selenium-webdriver';
+import { TestRunContext } from '../support/test-run-context';
+import { envConfig } from '../env-keys';
+
+export default class HomePage {
+  public static navigateToProfile(): Promise<void> {
+    return TestRunContext.browser.get(`${envConfig.HOMEPAGE_URL}/profile`);
+  }
+
+  public static getPageTitle(): By {
+    return By.css(`h1`);
+  }
+
+  public static getForm(form: string): By {
+    if (form === 'username') {
+      return By.css(`#card-username span.MuiCardHeader-title`);
```

Comment:
Be careful of `Mui` anything. Anything MUI is usually dynamically generated and could potentially change, though in this case might be fine. Just a note